src/core/mempool.rs

```diff
@@ -195,6 +196,24 @@ impl FromRow<MemPoolTxInfo> for MemPoolTxInfo {
     }
 }
 
+impl FromRow<(u64, u64)> for (u64, u64) {
+    fn from_row<'a>(row: &'a Row) -> Result<(u64, u64), db_error> {
+        let t1: i64 = row.get_unwrap(0);
+        let t2: i64 = row.get_unwrap(1);
+        if t1 < 0 || t2 < 0 {
+            return Err(db_error::ParseError);
+        }
+        Ok((t1 as u64, t2 as u64))
+    }
+}
+
+pub fn u32_to_sql(x: u32) -> Result<i32, Error> {
```

Comment:
This should not be necessary.  `ToSql` is already implemented for `i32` and `u32`.